interface-impl-amf/src/test/java/org/mule/amf/impl/parser/rule/ApiValidationResultImplTest.java

```diff
@@ -7,23 +7,34 @@
 
 package org.mule.amf.impl.parser.rule;
 
+import amf.client.validate.ValidationResult;
+import amf.core.annotations.LexicalInformation;
+import amf.core.parser.Position;
+import amf.core.parser.Range;
+import amf.core.validation.AMFValidationResult;
 import org.junit.Before;
 import org.junit.Test;
 import org.mule.amf.impl.AMFParser;
 import org.mule.apikit.model.api.ApiReference;
 import org.mule.apikit.validation.ApiValidationResult;
+import scala.Option;
 
 import java.util.List;
 import java.util.Optional;
 
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNull;
 
 public class ApiValidationResultImplTest {
 
     private static final String NO_VALID_RESOURCE_PATH = "not-valid-resource";
     private static final String ERROR_MESSAGE = "expected type: Integer, found: Double";
+    public static final String CUSTOM_ERROR_MESSAGE = "This is an error message!";
+    public static final String CUSTOM_LOCATION = "Here is the error";
```

Comment:
Can we add a URL to try the URLDecoder as well? Just to have an example of how it looks like